unittests/tools/test_ssh_audit_parser.py

```diff
@@ -0,0 +1,24 @@
+from ..dojo_test_case import DojoTestCase
+from dojo.tools.ssh_audit.parser import SSHAuditParser
+from dojo.models import Test
+
+
+class TestSSHAuditParser(DojoTestCase):
+
+    def test_parse_file_with_many_vuln_has_many_findings(self):
+        testfile = open("unittests/scans/ssh_audit/many_vulns.json")
+        parser = SSHAuditParser()
+        findings = parser.get_findings(testfile, Test())
+        for finding in findings:
+            for endpoint in finding.unsaved_endpoints:
+                endpoint.clean()
+        self.assertEqual(20, len(findings))
```

Comment:
It would be good to have some additional assertions here to further validate the parsing logic